angular-chart.js

```diff
@@ -185,6 +192,8 @@
               }, 50, false);
             }
             if (! scope.data || ! scope.data.length) return;
+            // Disable the existing watch. It has been installed before and it is installed again
+            cancel_watch_for_colours();
```

Comment:
Isn't the following code until installing the installer again executed sequentially as per JS event loop? In this case I don't really see how an other `watch` event could be triggered and interrupt the execution of the code.